### PR TITLE
Add Server 10.4.1 release notes

### PR DESCRIPTION
**Pull request by pmaier1**
As usual!

@micbar @phil-davis @settermjd

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: drone CI has passed: https://drone.owncloud.com/owncloud/docs/7560
There is currently some problem with web hooks to GitHub. The link and drone status are not getting back to the PR.

File: modules/admin_manual/pages/release_notes.adoc [hunk 1]
```diff
@@ -29,6 +30,26 @@
...
+* The `files:transfer-ownership` occ command can now also be executed for users who have never logged in. https://github.com/owncloud/core/pull/37038[#37038]
+* File download for files without a file extension from Google Drive external storages works now. https://github.com/owncloud/core/issues/37044[#37044]
+* The calculation of the remaining upload time in public links has been improved. https://github.com/owncloud/core/pull/37053[#37053]
+* E-mail notifications (e.g., for sharing) now respect the `default_language` config.php option. https://github.com/owncloud/core/issues/37039[#37039]
+* A new occ command (`files:check-cache`) is now available that allows to check if a target file can be read from the storage and to clean up stored information in the ownCloud's filecache in case a file disappears from the primary storage. This is mainly important for object stores and should only be utilized in rare cases. https://github.com/owncloud/core/pull/37067[#37067]
+
+Apart from this patch release, please consider the ownCloud Server 10.4.0 release notes, below.
```
Comment (micbar) [inline modules/admin_manual/pages/release_notes.adoc]: are both commas really needed?

Comment (settermjd) [reply modules/admin_manual/pages/release_notes.adoc:51]: With and without the second comma, the sentence is grammatically correct.

**Comment by phil-davis** [comment]
drone has passed https://drone.owncloud.com/owncloud/docs/7573/1/4
There are currently problems with drone reporting status back to GitHub. I will manually set the status.

**Comment by phil-davis** [comment]
If I merge this, then the proposed release notes will get published online to the docs.
Do we wait until when? There has been reasonable testing of RC1 or?

**Comment by settermjd** [comment]
> If I merge this, then the proposed release notes will get published online to the docs.
> Do we wait until when? There has been reasonable testing of RC1 or?

I agree that this PR waits until 10.4.1 is out of RC phase.

File: modules/admin_manual/pages/release_notes.adoc [hunk 2]
```diff
@@ -29,6 +30,28 @@
...
+* Improvements have been added to make long-running downloads more stable. https://github.com/owncloud/core/pull/36978[#36978]
+* Pending federated shares are now also shown in the "_Shared with you_" tab and can be accepted/declined there. https://github.com/owncloud/core/pull/37022[#37022]
+* The `files:transfer-ownership` occ command can now also be executed for users who have never logged in. https://github.com/owncloud/core/pull/37038[#37038]
+* File download for files without a file extension from Google Drive external storages now works. https://github.com/owncloud/core/issues/37044[#37044]
+* The calculation of the remaining upload time in public links has been improved. https://github.com/owncloud/core/pull/37053[#37053]
+* E-mail notifications (e.g., for sharing) now respect the `default_language` config.php option. https://github.com/owncloud/core/issues/37039[#37039]
+* A new occ command (`files:check-cache`) is now available. 
+  It checks if a target file can be read from the storage and to clean up stored information in ownCloud's filecache, in case a file disappears from the primary storage.
```
Comment (pmaier1) [inline modules/admin_manual/pages/release_notes.adoc:50]: [...] and **cleans** up [...] ?

Comment (phil-davis) [reply modules/admin_manual/pages/release_notes.adoc:50]: yes, that sounds better

**Comment by phil-davis** [comment]
@settermjd @pmaier1 https://github.com/owncloud-docker/server/pull/184#issuecomment-615091889
we are now at the point where people notice that the release notes are not yet published.

Merge this now and backport... ?

**Comment by pmaier1** [comment]
Yep, let's give this a go.

File: modules/admin_manual/pages/release_notes.adoc (hunk 2 above)
Comment (phil-davis) [reply modules/admin_manual/pages/release_notes.adoc]: fixed

**Pull request merged**